share/pvss/pvss.go

```diff
@@ -15,13 +15,28 @@ package pvss
 
 import (
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 	"github.com/dedis/crypto/proof"
-	"github.com/dedis/crypto/util/random"
 	"github.com/dedis/crypto/share"
+	"github.com/dedis/crypto/util/random"
 )
 
+// NOTE. Here, we need need the full suite definition for using the proof/
+// package while we only use Hash.
```

Comment:
Yes, it's a mapping of crypto-dependencies through the interfaces... And you get to see what exactly a certain package needs.